media/j2store/css/J4/j2store_admin.css

```
@@ -886,22 +869,11 @@ div.j2store-tab div.j2store-tab-content:not(.active){
         background-color:#132f53;
     }
     #j2store-navbar .img-circle {height:32px;width:auto;}
-    #j2store-navbar .navbar-nav .nav-item .nav-link {display: block;margin-right:0;width:100%;}
+    #j2store-navbar .navbar-nav .nav-item .nav-link {display: block;margin-inline-end:0;width:100%;}
```

```suggestion
    #j2store-navbar .navbar-nav .nav-item .nav-link {
      display: block;
      margin-inline-end:0;
      width:100%;
    }
```

consistent styling change. Just click on the button "commit suggestion"